integrations/Woopra/browser.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+// Research Spec: https://www.notion.so/rudderstacks/Woopra-Device-Mode-Research-Spec-adc482ccf8094965b539bce2453822a7
+
+import logger from '../../utils/logUtil';
+
+import { NAME } from './constants';
+
+class Woopra {
+  constructor(config) {
+    this.projectName = config.projectName;
+    this.name = NAME;
+  }
+
+  loadScript() {
+    !(function () {
+      var t,
+        o,
+        c,
+        e = window,
+        n = document,
+        r = arguments,
+        a = 'script',
+        i = [
+          'call',
+          'cancelAction',
+          'config',
+          'identify',
+          'push',
+          'track',
+          'trackClick',
+          'trackForm',
+          'update',
+          'visit',
+        ],
+        s = function () {
+          var t,
+            o = this,
+            c = function (t) {
+              o[t] = function () {
+                return o._e.push([t].concat(Array.prototype.slice.call(arguments, 0))), o;
+              };
+            };
+          for (o._e = [], t = 0; t < i.length; t++) c(i[t]);
+        };
+      for (e.__woo = e.__woo || {}, t = 0; t < r.length; t++)
+        e.__woo[r[t]] = e[r[t]] = e[r[t]] || new s();
+      ((o = n.createElement(a)).async = 1),
+        (o.src = 'https://static.woopra.com/w.js'),
+        (c = n.getElementsByTagName(a)[0]).parentNode.insertBefore(o, c);
+    })('Woopra');
+    window.Woopra.config({
+      domain: this.projectName, // This is the minimum required config
+    });
```

Comment:
what are the other config options and how it can be added?

Reply:
@anantjain45823 I do not see the addition configs are implemented over here. Let's add those as well